charts/ironic/values.yaml

```diff
@@ -406,7 +406,7 @@ endpoints:
         user_domain_name: service
         project_domain_name: service
       ironic:
-        role: admin,service
```

Comment:
We need a patch to add this.

Reply:
moved to a local patch

Reply:
https://review.opendev.org/c/openstack/openstack-helm/+/925931